### DISCUSSION TITLE
search / filter variables

### DISCUSSION THREAD

**Discussion by bsdgithub** [flowsforapex/apex-flowsforapex#721] [Ideas]
I have a process with more then 100 variables defined from the beginning. It would be useful to have a search or a filter to lookup for. Would that be an ideea for future development?

**Comment by rallen2010** [comment]
sounds like a good idea, but is this for the modeler or the flow monitor?  

In flow_monitor you can use the interactive report controls to search for a process variable currently.

**Comment by bsdgithub** [comment]
for the modeler, in the monitor, I can filter them, no problem
